diff --git a/build/config/compiler/compiler.gni b/build/config/compiler/compiler.gni
index d97ea02cc4..a4fffdf75b 100644
--- a/build/config/compiler/compiler.gni
+++ b/build/config/compiler/compiler.gni
@@ -46,7 +46,7 @@ declare_args() {
   cpp_standard = "gnu++14"
 
   # enable libfuzzer
-  is_libfuzzer = false
+  is_libfuzzer = true
 
   # Generate code coverage analysis artifacts when enabled.
   use_coverage = false
